streampipes-wrapper-python/streampipes/core.py

```diff
@@ -22,7 +22,10 @@
 from confluent_kafka.admin import AdminClient
 from confluent_kafka import Producer, Consumer
 
+from deprecated import deprecated
 
+
+@deprecated(version='0.93.0', reason="Please switch to StreamPipes Python (see README)")
```

Comment:
Can we mark this as deprecated for v0.92.0 and merge it into the 0.92.0 release branch?

Reply:
Damn, typo 
That was my intention 🙈